### DISCUSSION TITLE
TypeError [ERR_INVALID_ARG_TYPE]: The "url" argument must be of type string. Received undefined

### DISCUSSION THREAD

**Discussion by erkeen** [pactumjs/pactum#138] [Q&A] [answered]
When I invoke 
`spec.toss() `

it gives this error 
`TypeError [ERR_INVALID_ARG_TYPE]: The "url" argument must be of type string. Received undefined`

**Comment by ASaiAnudeep** [comment, marked as answer]
Please refer this discussion https://github.com/pactumjs/pactum/discussions/131

More details here - https://pactumjs.github.io/reference/toss
